wordpress-mobile/WordPress-iOS
# [Gravatar] - Add Gravatar info UI in user profile settings

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/22551

We are looking into how we can make our users aware that when editing a profile, they are actually editing their Gravatar profile. This PR adds some informative UI for that purpose.

**Jetpack:**

<table>
  <tr>
    <td><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/55b17a56-a278-42be-a47d-bc520002f1a5" width=300/></td>
    <td><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/d2402f13-9e53-4dff-b195-fb66b0240f0e" width=300/></td>
  </tr>
</table>

**WordPress:**

<table>
  <tr>
    <td><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/9a0f2a1f-0aed-4472-b02c-1840c5a3944b" width=300/></td>
    <td><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/271f1daa-7df0-426c-aaa5-ed018932820a" width=300/></td>
  </tr>
</table>

To test:

Try on both JP and WP:

- Go to the profile tab > My Profile
- Observe: There's a new informative UI as demonstrated above ☝️ 
- Try updating those 4 fields
- Observe you see a notice message saying "Updates might take some time to sync with your Gravatar profile."
 
(There's a possibility that we use a different copy for the notice message. But it should not be a blocker for reviewing the PR. Just letting you know that I might push a new commit for this.)

## Regression Notes

1. Potential unintended areas of impact

Account settings. App settings. This time the notice should NOT show upon updating fields.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

I tried changing some fields in Account settings. App settings. Verified that the notice in NOT shown in that case.

3. What automated tests I added (or what prevented me from doing so)

I looked into writing some snapshot tests but this repo doesn't have any of those. That requires a new snapshot library added to the repo and that would be out of context of this PR.

PR submission checklist:


Testing checklist:
